### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewOutput_ScreenReader: NVDA does not announce any information when the state of Toggle Output Scroll lock button changes.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52184]
Environment Details:
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2

Repro Steps:
1. Launch VS Code while NVDA is ON.
2. Go to View menu and Select Output.

Actual:
NVDA does not announce any information when the state of Toggle Output Scroll lock button changes. Visually its icon changes when we click over it, Once it gets lock and after another click it gets unlock.

Expected:
NVDA should read any infomation when the state of toggle output button changes, about its functionality or what it does and also it should announce state change(lock/unlock)

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not be able to know the use of that button or the state of that button.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for reference: 
[A11y VSCode Output NVDA Scroll Lock button.pptx](https://github.com/Microsoft/vscode/files/2110993/A11y.VSCode.Output.NVDA.Scroll.Lock.button.pptx)

Does this issue occur when all extensions are disabled?: Yes
